harvester/projects/tests/seeding/test_buas.py

```diff
@@ -0,0 +1,125 @@
+from pathlib import Path
+
+from django.conf import settings
+from django.test import TestCase
+
+from datagrowth.resources.testing import ResourceFixturesMixin
+
+from core.constants import DeletePolicies
+from core.processors import HttpSeedingProcessor
+from testing.cases import seeding
+from projects.models import Set, BuasPureProjectResource
+from projects.sources.buas import SEEDING_PHASES
+
+
+class TestBuasProjectSeeding(seeding.ResourceFixturesSeedingTestCase):
+
+    fixtures_directory = Path(settings.BASE_DIR, "projects", "fixtures", "resources", "buas")
+    resource_fixtures = ["buas-test"]
+    delta_fixtures = {
+        (BuasPureProjectResource, 1): ("body", "buas-projects.02.pii.json")
```

Comment:
Hier geven we aan dat tijdens deltas (de niet initiele harvest, die waarschijnlijk een subset van de data is) een andere PII data file gebruikt moet worden. Hiermee simuleren we creates, updates en deletes voor in dit geval BUAS, maar elk bronsysteem heeft daarin natuurlijk eigen mechanismen.